src/Posts.js

```diff
@@ -1,31 +1,33 @@
-import React from 'react'
-import axios from 'axios'
+import React, { useState, useEffect } from "react";
 
-export default class Posts extends React.Component {
-    state = {
-        posts: []
-    }
+export default function Posts(props) {
+    const [posts, setPosts] = useState([
+        {
+            posts: []
+        }
+    ]);
 
-    componentDidMount() {
-        axios.get('http://0.0.0.0:3000/posts.json')
-            .then(res => {
-                const posts = res.data;
-                this.setState({ posts });
-            })
+    async function fetchPostsData() {
+        const response = await fetch(
+            `http://0.0.0.0:3000/posts.json`
```

Comment:
`0.0.0.0` is a broadcast address? I assume this is meant to be `localhost`? Either way, you should generally avoid hard coded URLs in code and inject them from config, or dynamically construct them on the client side, e.g. by using `window.location` and its sub-propoerties

Reply:
I know, I've got the same feedback from Tony. This PR is only to fix the issue I was having before. I plan to change that but I'll have to change it later.